docs/sim-rules.md

```diff
@@ -112,16 +115,15 @@ A deployable that repairs itself will NEVER be allowed in a Chaos Sim.
 1. Standard damage enabled projectiles (Bullets) must die on collision with solid surfaces.
 2. Projectile Rotations should be locked through `llSetStatus(STATUS_ROTATE_X|STATUS_ROTATE_Y|STATUS_ROTATE_Z,0);` on any non arcing munition, excluding seekers. 
     * Munitions that arc through the air, ie grenades, mortars, etc, should have X and Z locked regardless. This is to prevent projectiles from wildly ricocheting.
-3. Projectile Density should be set to 1000 or lower, to prevent bullets from bouncing physical vehicles.
-4. Munitions must be fired along a interceptable trajectory. Under no circumstances can something warp from position A to position B with something between it.
-5. Explosive munition hitboxes should be no smaller than 0.15 meters, height and widthwise, non-phantom, and should be able to be countered by all interception systems. Don't try being a sneaky little faggot.
-6. Fully automatic explosive bullets (FAEB) must be fired from an appropriate platform. These are the rules for the PROJECTILES.
-    * Man portable FAEB weapons should fire at a relatively low rate of fire (Less than 400 RPM for 'Rifles', Less than 700 for 'LMG's), and may not be capable of AT.
-    * Vehicle mounted weapons have a higher rate of fire, according to the mounted weapon. Only FIXED, AIMED BY TURNING THE ENTIRE VEHICLE, or FLEETSHIP mounted FAEBs may do DIRECT HIT AT at a MAXIMUM of 5 AT. AT should scale with RPM, naturally.
-7. Fully automatic AT bullets (FAAB) must be fired from an appropriate platform.
-    * Man portable FAAB weapons should fire at an EXTREMELY low rate of fire (Less than 300 RPM, a sacrifice is expected for AT output at automatic rates) and may do DIRECT HIT AT at a **MAXIMUM** of 5 AT. In that regard, 5 AT Should cap at 60 RPM, scale from there.
-    * Man portable FAAB weapons MAY NOT also have an explosion.
-    * Vehicle mounted FAAB munitions may do DIRECT HIT AT at a MAXIMUM of 5 AT. Only FIXED, AIMED BY TURNING THE ENTIRE VEHICLE, or FLEETSHIP mounted FAABs may explode.
+3. **Explosive munitions should be no smaller than 0.06 meters, height and widthwise, non-phantom, and should be able to be countered by all interception systems. Low yield explosive bullets are the only exception, see #6 of this section.**
```

Comment:
Make this Explosive **Projectiles** not Munitions

Reply:
yes